src/index.html

```diff
@@ -0,0 +1,452 @@
+      </div>
+      <div>
+        <img src="./img/3.png" class="w-[66.44px] mb-[20px]" />
+        <p class="text-[32px] font-[700] leading-[120%] mt-[10px] mb-[20px]">
+          High security to protect from piracy
+        </p>
+        <p class="font-[400] text-[16px] opacity-[0.87] mt-[10px]">
+          Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam
+          nonumy.
+        </p>
+      </div>
+      <div>
+        <img src="./img/3.png" class="w-[66.44px] mb-[20px]" />
+        <p class="text-[32px] font-[700] leading-[120%] mt-[10px] mb-[20px]">
+          High security to protect from piracy
+        </p>
+        <p class="font-[400] text-[16px] opacity-[0.87] mt-[10px]">
+          Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam
+          nonumy.
+        </p>
+      </div>
+    </div>
+    <section
+      class="mr-[5%] ml-[5%] mt-[120px] gap-[40px] grid grid-cols-2 mb-[100px]"
+    >
+      <div class="m-auto">
+        <p
+          class="text-[24px] font-semibold bg-clip-text text-transparent bg-gradient-to-r from-[#373FFF] to-[#3ACAF8] mb-[20px]"
+        >
+          Why should you work with us?
+        </p>
+        <p
+          class="text-[56px] font-[600] leading-[110%] mb-[20px] text-[#1D2130]"
+        >
+          To upscale your business to the next level
+        </p>
+        <p class="text-[16px] leading-[150%] font-[400]">
+          Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam
+          nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat,
+          sed diam voluptua invidunt ut labore.
+        </p>
+      </div>
+      <div class="m-auto">
+        <div class="flex gap-[20px] mb-[30px]">
+          <span
+            class="bg-[#FFACAC] h-[48px] w-[48px] mt-auto mb-auto p-[17px] rounded-[50%]"
+            ><img src="./img/Arrow.png"
+          /></span>
+          <p class="text-[20px] font-[400] leading-[150%]">
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+            eiusmod.
+          </p>
+        </div>
+        <div class="flex gap-[20px] mb-[30px]">
+          <span
+            class="bg-[#FFACAC] h-[48px] w-[48px] mt-auto mb-auto p-[17px] rounded-[50%]"
+            ><img src="./img/Arrow.png"
+          /></span>
+          <p class="text-[20px] font-[400] leading-[150%]">
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+            eiusmod.
+          </p>
+        </div>
+        <div class="flex gap-[20px] mb-[30px]">
+          <span
+            class="bg-[#FFACAC] h-[48px] w-[48px] mt-auto mb-auto p-[17px] rounded-[50%]"
+            ><img src="./img/Arrow.png"
+          /></span>
+          <p class="text-[20px] font-[400] leading-[150%]">
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+            eiusmod.
+          </p>
+        </div>
+      </div>
+    </section>
+    <div class="more-info__background"></div>
```

Comment:
`more-info__background` is an element of `more-info`. where is the `more-info` block.